msm/src/verifier.rs

```diff
@@ -50,25 +51,24 @@ pub fn verify<
     let omega = domain.d1.group_gen;
     let zeta_omega = zeta * omega;
 
-    let mut es: Vec<_> = zeta_evaluations
+    let mut es: Vec<Vec<Vec<G::ScalarField>>> = zeta_evaluations
```

Comment:
We will change the type later. But it does imply changes outside the scope of this PR